routes/v1.js

```diff
@@ -156,7 +160,7 @@ api.declare({
   name:         'getHookSchedule',
   output:       'hook-schedule.json',
   title:        'Get hook schedule',
-  stability:    'deprecated',
+  stability:    'experimental',
```

Comment:
This one should stay at "deprecated" - the information it provides is now available from `getHookStatus`.